lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -549,6 +557,11 @@ abstract class DefaultTextEditingStrategy implements TextEditingStrategy {
   final List<StreamSubscription<html.Event>> _subscriptions =
       <StreamSubscription<html.Event>>[];
 
+  bool get hasAutofillGroup => _inputConfiguration.autofillGroup != null;
+
+  html.FormElement focusedFormElement() =>
```

Comment:
Is there a reason you made this a method and not a getter?

Reply:
done. thanks for the suggestion.